include/dali/core/cuda_error.h

```diff
@@ -12,12 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef DALI_CORE_CUDA_ERROR_H
-#define DALI_CORE_CUDA_ERROR_H
+#ifndef DALI_CORE_CUDA_ERROR_H_
+#define DALI_CORE_CUDA_ERROR_H_
 
 #include <cuda_runtime_api.h>
 #include <cstring>
 #include <cstdio>
+#include <string>
```

Comment:
Could you add a new line to separate include files groups (system and project includes)? It allows to sort them using clang-format.